packages/components/src/helpers/propsToCssVariables.ts

```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type CSSProperties } from 'react';
+
+import { type Breakpoint } from './layout';
+
+const abbreviationToCssProperty: Record<string, string> = {
+    m: 'margin',
+    mx: 'margin-x',
+    my: 'margin-y',
+    mt: 'margin-top',
+    mr: 'margin-right',
+    mb: 'margin-bottom',
+    ml: 'margin-left',
+    p: 'padding',
+    px: 'padding-x',
+    py: 'padding-y',
+    pt: 'padding-top',
+    pr: 'padding-right',
+    pb: 'padding-bottom',
+    pl: 'padding-left',
+    direction: 'flex-direction',
+    align: 'align-items',
+    wrap: 'flex-wrap',
+    columns: 'grid-template-columns',
```

Comment:
not sure if this is smart, maybe it makes more sense to call this "column-template" or something.

Because it supports a template string an not only a number

Reply:
Added support for `columns={3}` or `columns="repeat(3, 1fr)"`